Tools/autotest/sim_vehicle.py

```diff
@@ -851,6 +852,11 @@ def start_mavproxy(opts, stuff):
     location = find_location_by_name(find_autotest_dir(), cmd_opts.location)
     progress("Starting up at %s (%s)" % (location, cmd_opts.location))
 
+if cmd_opts.use_dir is not None:
+    new_dir = cmd_opts.use_dir
+    os.makedirs(new_dir)
```

Comment:
Ok, after looking more at the documentation it looks like we could differentiate them by the error number so I agree to catch the exception.